test/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+###############################################################################
+# Copyright (c) 2017-22, Lawrence Livermore National Security, LLC
+# and RAJA Performance Suite project contributors.
+# See the RAJAPerf/LICENSE file for details.
+#
+# SPDX-License-Identifier: (BSD-3-Clause)
+###############################################################################
+
+include_directories(../src)
```

Comment:
I think this should be a `target_include_directories` instead, e.g.:
```
target_include_directories(test-raja-perf-suite.exe PRIVATE ../src)
```

You could also use `PROJECT_SOURCE_DIR/src` to avoid using the relative path.

Reply:
Using `PROJECT_SOURCE_DIR/src` the test doesn't build since it can't find the needed header files.  

`target_include_directories(test-raja-perf-suite.exe PRIVATE ../src)`  works if I put it after the `raja_add_test` macro call

Reply:
Besides being more verbose, what is the advantage of this over what I had originally?

Reply:
`${PROJECT_SOURCE_DIR}/src` should work. It's mostly a best practice thing.

Reply:
change pushed to use target_include_directories